app/javascript/modules/mixer.js

```diff
@@ -241,3 +243,13 @@ function loadActionCableSlidersMaster() {
 }
 
 // --
+
+export {
+  loadActionCableSliders,
+  loadActionCableSlidersMaster,
+  rangeSlider,
+  rangeSliderMaster,
+  resetRangeSliderValue,
+  resetRangeSliderValueMaster,
+  toggleButton,
+};
```

Comment:
I didn't rewrite anything in this file (which used to be called [`channels_index.js`](https://github.com/vindennl48/HipBox/blob/d6d1f128d7ca808884a542dc94f0e8594b51bf41/app/assets/javascripts/channels_index.js)), but I did have to export a lot of stuff. I think this file is probably a good candidate for refactoring into individual ActionCable channels.

Reply:
If you take a look at my feature branch `mw-revamp_js_and_views`, I overhauled all of this garbage JS anyway.

Reply:
At first glance, it looks much cleaner – I think I'd find it helpful to talk through some of the JS, if we can find the time.